api/common.js

```diff
@@ -413,6 +416,125 @@ exports.load_github_detail = function(service_name, cb) {
     }).catch(cb)
 }
 
+exports.mag_evaluate = function(query, projectid){
```

Comment:
This function makes query to MAG, massage the output, then update project record. I suggest renaming it to something like` updateProjectMag` or `updateProjectRelatedPapers`, etc..